src/inference/src/model_reader.cpp

```diff
@@ -14,6 +14,20 @@
 #include "transformations/utils/utils.hpp"
 
 namespace {
+// These deperecated functions are moved here as static to facilitate for IR v10
+static inline std::string create_ie_output_name(const ov::Output<const ov::Node>& output) {
```

Comment:
Can we avoid duplicates in many files?

Reply:
Yeah, I also didn't like it. I've removed the functions replacing their calls with what what they actually do. Thankfully, they are not that big. No functions, no problems :)